app/reducers/comments.ts

```diff
@@ -9,10 +9,11 @@ export type CommentEntity = {
   id: ID;
   parent?: ID;
   createdAt: string;
-  children: Array<Record<string, any>>;
+  children?: Array<Record<string, any>>;
```

Comment:
Could you type this better while you're at it :))

Reply:
I don't really want to :grimacing:.

Trying to focus on the TS errors, and typing the `any`s that are pretty easy. This one I don't really know what should be so I don't want to spend a lot of time on it.